Dockerfile

```diff
@@ -1,119 +1,37 @@
-                   certmonger              \
-                   sssd                    \
-                   sssd-dbus               \
-                   c-ares                  \
-                   real-md                 \
-                   adcli                   \
-                   oddjob                  \
-                   oddjob-mkhomedir        \
-                   realmd                  \
-                   samba-common            \
                    &&                      \
     yum clean all
 
-# Systemd cleanup base image
-RUN (cd /lib/systemd/system/sysinit.target.wants && for i in *; do [ $i == systemd-tmpfiles-setup.service ] || rm -vf $i; done) && \
-     rm -vf /lib/systemd/system/multi-user.target.wants/* && \
-     rm -vf /etc/systemd/system/*.wants/* && \
-     rm -vf /lib/systemd/system/local-fs.target.wants/* && \
-     rm -vf /lib/systemd/system/sockets.target.wants/*udev* && \
-     rm -vf /lib/systemd/system/sockets.target.wants/*initctl* && \
-     rm -vf /lib/systemd/system/basic.target.wants/* && \
-     rm -vf /lib/systemd/system/anaconda.target.wants/*
-
-## GIT clone manageiq-appliance and service UI repo (SUI)
-RUN mkdir -p ${APPLIANCE_ROOT} && \
-    curl -L https://github.com/ManageIQ/manageiq-appliance/tarball/${REF} | tar vxz -C ${APPLIANCE_ROOT} --strip 1
-RUN mkdir -p ${SUI_ROOT} && \
-    curl -L https://github.com/ManageIQ/manageiq-ui-service/tarball/${REF} | tar vxz -C ${SUI_ROOT} --strip 1
-
-## Setup environment
-RUN ${APPLIANCE_ROOT}/setup && \
-    rm -f /etc/httpd/conf.d/manageiq-balancer-* && \
-    ln -vs ${APP_ROOT} /opt/manageiq/manageiq && \
-    mkdir -p ${APP_ROOT}/log && \
-    mkdir -p ${APP_ROOT}/public && \
-    ln -s /persistent-assets/assets ${APP_ROOT}/public/assets && \
-    mkdir -p ${CONTAINER_SCRIPTS_ROOT} && \
-    mv /etc/httpd/conf.d/ssl.conf{,.orig} && \
-    echo "# This file intentionally left blank. ManageIQ maintains its own SSL configuration" > /etc/httpd/conf.d/ssl.conf 
+## Remove any existing configurations
+RUN rm -f /etc/httpd/conf.d/*
 
-## Change workdir to application root, build/install gems
-WORKDIR ${APP_ROOT}
-
-## Build SUI
-RUN cd ${SUI_ROOT} && \
-    npm install yarn -g && \
-    yarn install --production && \
-    yarn run build && \
-    yarn cache clean
-
-## Expose required container ports
-EXPOSE 80 443
-
-## Copy Apache configuration files
-COPY docker-assets/apache-conf/*.conf /etc/httpd/conf.d/
-
-## Copy OpenShift and appliance-initialize scripts
 COPY docker-assets/entrypoint /usr/bin
-COPY docker-assets/container.data.persist /
-COPY docker-assets/appliance-initialize.sh /bin
-ADD  docker-assets/container-scripts ${CONTAINER_SCRIPTS_ROOT}
+COPY docker-assets/generate_server_cert.sh /
+COPY docker-assets/manageiq.conf /etc/httpd/conf.d/
 
-RUN systemctl enable dbus httpd
+RUN chmod +x /generate_server_cert.sh
 
-VOLUME [ "/sys/fs/cgroup" ]
+EXPOSE 80 443
 
 ENTRYPOINT [ "entrypoint" ]
-CMD [ "/usr/sbin/init" ]
+CMD ["/run-httpd.sh"]
```

Comment:
Does this work without systemd?

I thought `apachectl` used systemd under the covers.